src/core/xconnection.rs

```diff
@@ -811,117 +855,121 @@ impl XConn for XcbConnection {
 
     fn set_wm_properties(&self, workspaces: &[&str]) {
         // xcb docs: https://www.mankier.com/3/xcb_change_property
+        xcb::change_property(
```

Comment:
I'm sorry if the diff looks a bit confusing here, I'm guessing it's because of the formatting changes. All I did here was replace `known_atom()` with `*self.atoms.get().unwrap()`.